src/Form/Form.js

```diff
@@ -485,14 +493,18 @@ class Form {
                 return
             }
 
-            const input = e.target
+            // On mobile, we don't trigger on focus, so here we get the target control on label click
+            const isLabel = e.target instanceof HTMLLabelElement
+            const input = isLabel ? e.target.control : e.target
+            if (!input || !this.inputs.all.has(input)) return
```

Comment:
This is new. It's not needed on desktop because we rely on the `focus` event, but we can't do that on mobile to avoid keyboard flashing. This is a simple workaround.